src/summary/ScatterPlotVisualizer.jsx

```diff
@@ -36,9 +45,18 @@ class ScatterPlotVisualizer extends Component {
         return [alive, dead];
     }
 
+    resize = () => {
+        this.setState({
+            chartHeight: "30px"
+        })
+    }
+
     renderScatterPlot = (patient, condition, conditionSection) => {
 
         const myData = conditionSection.data[0].data_cache;
+        if (myData === "Server unavailable" || myData === "No relevant data found for patient") {
```

Comment:
[GQ1] In case the service client adds additional error messages, we should code this part to check if the data_cache is a string. If it's a string, it's an error. If it's a JSON object, it's actual data to graph. We can add a comment to make that clear here as well.